app/components/player/kick-player.tsx

```diff
@@ -0,0 +1,79 @@
+import { useFetcher } from "@remix-run/react";
+
+import type { Action, Player } from "~/engine";
+import { useEngineContext } from "~/engine";
+import useSoloAction from "~/utils/use-solo-action";
+import { PlayerScoreBox } from "./player";
+
+function KickPlayer({
+  hasBoardControl,
+  player,
+  winning = false,
+}: {
+  hasBoardControl: boolean;
+  player: Player;
+  winning?: boolean;
+}) {
+  return (
+    <PlayerScoreBox player={player} hasBoardControl={hasBoardControl}>
+      <div className="flex w-full items-center gap-2 text-2xl">
+        <p className="font-handwriting font-bold text-slate-300">
+          {player.name}
+        </p>
+        <div className="ml-auto flex items-center gap-2">
+          <button
```

Comment:
oh, can you also add alt text to this that shows up on hover like `Kick player ${player.name}`